common/src/View/CompilationProfileManager.cpp

```diff
@@ -100,13 +110,16 @@ namespace TrenchBroom {
             if (m_config.profileCount() == 1) {
                 m_profileList->SetSelection(wxNOT_FOUND);
                 m_config.removeProfile(static_cast<size_t>(index));
+				m_config.setLastUsedProfileIndex(-1);  // Remember last selection index this session -Qmaster
             } else if (index > 0) {
                 m_profileList->SetSelection(index - 1);
                 m_config.removeProfile(static_cast<size_t>(index));
+				m_config.setLastUsedProfileIndex(index - 1);  // Remember last selection index this session -Qmaster
```

Comment:
This is only correct if the last used profile index is always the same as the selected index, is that sure? Also please remove all these comments, they are really useless since we can trace the changes with git blame.